src/EasyAdminBundle/Controller/AdminControllerPreventingFromSelfDelete.php

```diff
@@ -1,14 +1,14 @@
 <?php
 
-namespace AppBundle\Controller\Admin;
+namespace EasyAdminBundle\Controller;
 
 use Doctrine\DBAL\Exception\ForeignKeyConstraintViolationException;
-use JavierEguiluz\Bundle\EasyAdminBundle\Controller\AdminController as BaseAdminController;
+use JavierEguiluz\Bundle\EasyAdminBundle\Controller\AdminController as AdminController;
```

Comment:
This alias is redundant. This line should be just `use JavierEguiluz\Bundle\EasyAdminBundle\Controller\AdminController`. It'll work.